assets/sass/components/key-metrics/_googlesitekit-chip-tab-group.scss

```diff
@@ -18,25 +18,63 @@
 
 .googlesitekit-plugin {
 	.googlesitekit-chip-tab-group {
+		%chip-item {
+			background-color: $c-neutral-n-50;
+			color: $c-neutral-n-900;
+			font-size: $fs-label-sm;
+			font-weight: $fw-medium;
+			min-height: 28px;
+			padding: 6px 14px;
+
+			--mdc-ripple-fg-opacity: 0;
+		}
 
 		.googlesitekit-chip-tab-group__tab-items {
+			background: #fff;
 			display: flex;
-			flex-wrap: wrap;
+			flex-wrap: nowrap;
 			gap: 12px;
-			margin: 0 25px 25px;
+			left: 0;
+			margin: 0 16px 15px;
+			overflow: auto;
+			position: sticky;
+			top: 0;
+			white-space: nowrap;
+			z-index: 1;
 
-			@media (min-width: $bp-desktop) {
-				flex-direction: column;
+			@media (min-width: $bp-tablet) {
+				flex-wrap: wrap;
+				margin: 0 25px 25px;
+				overflow: unset;
+				white-space: unset;
 			}
+
+			/** Mobile version style adaptation - start **/
+			.mdc-tab-scroller__scroll-content {
+				gap: 10px;
+			}
+
+			.mdc-tab {
+				@extend %chip-item;
+				height: unset;
+				line-height: unset;
+
+				&.mdc-tab--active {
+					background-color: $c-site-kit-sk-100;
+				}
+
+				.mdc-tab-indicator {
+					> .mdc-tab-indicator__content {
+						display: none;
+					}
+				}
+			}
+
+			/** Mobile version style adaptation - end **/
```

Comment:
How are these mobile-specific adaptations if they aren't scoped to a particular breakpoint?

Reply:
These classes are rendered by `TabBar` component, which is rendered only on mobile to produce smooth horizontal scroll, indicated in [this comment](https://github.com/google/site-kit-wp/pull/9661/files#r1837180345)